client/src/Components/Layouts/Ordersmanagement/index.jsx

```diff
@@ -1,23 +1,41 @@
 import React, { Component } from "react";
 import axios from "axios";
+import { DatePicker, Input, Button, Icon } from "antd";
+import moment, { isValid } from "moment";
+import Sidebar from "../../CommonComponent/Sidebar/index";
+import Header from "../../CommonComponent/Header/index";
+import Navbar from "../../CommonComponent/Navbar/index";
 import TableComponent from "../../CommonComponent/Table/Table";
 import {
-  viewPopup,
-  editPopup,
+//   viewPopup,
+//   editPopup,
   deletePopup
 } from "../../CommonComponent/Table/Popups";
+import "./style.css";
 
 class OrdersManagement extends Component {
   state = {
     orders: [],
-    error: ""
+    filteredOrders: [],
+    date: "",
+    status: "",
+    name: "",
+    error: "",
+    filter: false
   };
 
   componentDidMount() {
     axios
       .get("/api/v1//viewOrders")
       .then(res => {
-        this.setState({ orders: res.data });
+        if (res.status === 204) {
```

Comment:
no need to check if there is content or not, the empty table doesn't make any problem